diesel_cli/src/cli.rs

```diff
@@ -182,6 +182,17 @@ pub fn build_cli() -> App<'static, 'static> {
         .global(true)
         .takes_value(true);
 
+    let verify_schema_arg = Arg::with_name("VERIFY_SCHEMA")
+        .long("verify-schema")
+        .help("Verify that the schema file is unchanged")
```

Comment:
Hmmmm, not sure about the name. I'd expect this to _additionally_ verify the schema, not to error out. Cargo does something similar but calls it `--locked` ("Require Cargo.lock is up to date") and `--frozen` ("Require Cargo.lock and cache are up to date").

Reply:
I do want to mention *what* file it's looking at here -- What do you think about `--locked-schema`?

Reply:
SGTM!